xblocks_contrib/poll/poll.py

```diff
@@ -1,31 +1,64 @@
 """TO-DO: Write a description of what this XBlock is."""
 
-from importlib.resources import files
+import uuid
+import html
+import json
 
+from importlib.resources import files
+from copy import deepcopy
+from collections import OrderedDict
 from django.utils import translation
+from lxml import etree
 from web_fragments.fragment import Fragment
 from xblock.core import XBlock
-from xblock.fields import Integer, Scope
+from xblock.fields import Boolean, Dict, List, Scope, String
 from xblock.utils.resources import ResourceLoader
+# from xblocks_contrib.poll.xml_block import XmlMixin
+# from xblocks_contrib.poll.x_module import XModuleMixin
+from xmodule.x_module import XModuleMixin
+from xmodule.xml_block import XmlMixin
+from openedx.core.djangolib.markup import Text, HTML
 
+
+_ = lambda text: text
 resource_loader = ResourceLoader(__name__)
 
 
-# This Xblock is just to test the strucutre of xblocks-contrib
 @XBlock.needs("i18n")
-class PollBlock(XBlock):
-    """
-    TO-DO: document what your XBlock does.
-    """
-
-    # Fields are defined on the class.  You can access them in your code as
-    # self.<fieldname>.
+class PollBlock(XmlMixin, XModuleMixin):
```

Comment:
As this xblock do not have studio view so we need to add the data in OLX and the xblock read the data from XML when we export the course. 
we override the methods from XmlMixin to parse the data from XML, this mixin is in xmodule if we extent like this the data load properly but without that it is not reading the data from xml. 
According to my information, there are a few mixins that are incorporated when the block is initialized. We can see the code in `common.py` of `edx-platform`; there is `XBLOCK_MIXINS`, but for some reason, it's not working.